projects/core/src/occ/adapters/cart/occ-cart-entry.adapter.ts

```diff
@@ -32,6 +32,7 @@ export class OccCartEntryAdapter implements CartEntryAdapter {
       {
         userId,
         cartId,
+        quantity,
       },
       { code: productCode, qty: quantity }
```

Comment:
how about this: `{ code: productCode, qty: quantity, quantity: quantity }`. So, no need change the endpoint config. (we pass `qty` and `quantity` both :)

Reply:
Both approaches work well.  The slight advantage with the quantity parameter in the b2b endpoint config is that it will not appear in the b2c requests.  Of course we still get both qty and quantity when using b2b.